Common/app.config

```diff
@@ -67,19 +67,10 @@
       <setting name="ActivityWindow_Orientation" serializeAs="String">
         <value>Horizontal</value>
       </setting>
-      <setting name="StartNotifierAfterLogin" serializeAs="String">
-        <value>True</value>
-      </setting>
-      <setting name="StartNotifierMinimized" serializeAs="String">
-        <value>False</value>
-      </setting>
-      <setting name="IsInstalled" serializeAs="String">
-        <value>False</value>
-      </setting>
-    </Wokhan.WindowsFirewallNotifier.Common.Settings>

```

Comment:
These are the settings which where introduced in master - need to check whether they are still (again) there....

Reply:
Ok, the new settings are still there.

Reply:
Ok so I'm even more disappointed. I added them back when resolving the conflicts, so other modifications should have been there also...